build/runTests.ps1

```diff
@@ -0,0 +1,127 @@
+function runTests() {
```

Comment:
This first method was moved from the existing _build.ps1_ file.
I added the _/EnabledCodeCoverage_ and _/Logger_ arguments.